protocols/kad/src/behaviour/test.rs

```diff
@@ -755,7 +756,7 @@ fn get_record() {
         .map(|(_addr, swarm)| swarm)
         .collect::<Vec<_>>();
 
-    let record = Record::new(random_multihash(), vec![4, 5, 6]);
+    let record = Record::new(random_multihash(), Bytes::from(vec![4, 5, 6]));
```

Comment:
For these tests, we can use `from_static`!

```suggestion
    let record = Record::new(random_multihash(), Bytes::from_static(&[4, 5, 6]));
```